server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -321,7 +325,7 @@ Issues related to authorization may exist just for some other versions of {jdgse
                 ...
 ```
 +
-It is currently required to have different configuration files for the JDG servers on both sites as the Infinispan subsystem does not support replacing site names with expressions. See link:https://issues.redhat.com/browse/WFLY-9458[this issue] for more details.
+It is currently required to have different configuration files for the {jdgserver_name} servers on both sites as the Infinispan subsystem does not support replacing site names with expressions. See link:https://issues.redhat.com/browse/WFLY-9458[this issue] for more details.
```

Comment:
Data Grid 8.x is no longer based on WildFly/EAP. It doesn't make sense to talk about the Infinispan subsystem. 

I would also remove the link to that issue because it's closed. Maybe replace it with https://issues.redhat.com/browse/WFLY-7871 

IMHO this isn't something we should document as a limitation. That issue is an enhancement suggested by engineering and shouldn't be documented as though it were a bug/issue with Infinispan.

Reply:
I would remove this. As Don said, it is no longer valid for ISPN 11 / DG8

Reply:
this is fixed.